connectors/sources/tests/fixtures/sharepoint/fixture.py

```diff
@@ -13,11 +13,25 @@
 from flask import Flask, request
 
 app = Flask(__name__)
-start_lists, end_lists = 0, 450
+start_lists, end_lists = 0, 4500
 DATA_SIZE = os.environ.get("DATA_SIZE", "small").lower()
 _SIZES = {"small": 1000000, "medium": 2000000, "large": 6000000}
 FILE_SIZE = _SIZES[DATA_SIZE]
 LARGE_DATA = "".join([random.choice(string.ascii_letters) for _ in range(FILE_SIZE)])
+DOC_ID_SIZE = 36
+
+def adjust_document_id_size(id):
+    """
+      This methods make sure that all the documemts ids are min 36 bytes like in Sharepoint
+    """
+
+    bytesize = len(id)
+
+    if bytesize >= DOC_ID_SIZE:
+        return id
+
+    addition = "".join([random.choice(string.ascii_letters) for _ in range(DOC_ID_SIZE-bytesize)])
```

Comment:
Do you need to make sure the same document always has the same id? considering you may need to execute multiple syncs in the test?

Reply:
Yes, you are right. It should not be `random`. Good catch!